test/server/routes/__mocks__/userData.js

```diff
@@ -120,3 +120,28 @@ export const wrongLengthInput = {
   password: 'veryveryverylongpasswordhere',
   avatarUrl: 'https://gravatar.com/avatar/aed61baf1e9256ed7d70e2cbbfcba9aa?s=400&d=robohash&r=x',
 };
+
+export const patronLogin = {
+  email: 'johnsimons@gmail.com.ng',
+  password: 'g4g5j6lkl4k9',
+};
+
+export const newBook = {
```

Comment:
Isn't it better for the data relating to books to be in a separate file, say 'bookData' ?

Reply:
I put it here because it relates to the user controller and user route.